CHANGELOG.md

```diff
@@ -1,23 +1,72 @@
 x.y.z Release notes (yyyy-MM-dd)
 =============================================================
-### Enhancements
-* None.
 
-### Fixed
-* <How to hit and notice issue? what was the impact?> ([#????](https://github.com/realm/realm-swift/issues/????), since v?.?.?)
-* None.
+This version bumps the Realm file format version. Realm files written by this
+version cannot be read by older versions of Realm.
+
+### Enhancements
+* The Realm file is now automatically shrunk if the file size is larger than
+  needed to store all of the data. ([Core PR #5755](https://github.com/realm/realm-core/pull/5755))
+* Pinning old versions (either with frozen Realms or with Realms on background
+  threads that simply don't get refreshed) now only prevents overwriting the
+  data needed by that version, rather than the data needed by that version and
+  all later versions. In addition, frozen Realms no longer pin the transaction
+  logs used to drive change notifications. This mostly eliminates the file size
+  growth caused by pinning versions. ([Core PR #5440](https://github.com/realm/realm-core/pull/5440))
+* Rework how Dictionaries/Maps are stored in the Realm file. The new design uses
+  less space and is typically significantly faster. This changes the iteration
+  order of Maps, so any code relying on that may be broken. We continue
+  to make no guarantees about iteration order on Maps.
+* Improve performance of acquiring read locks when a read lock for that version
+  is already held. This speeds up many operations related to change
+  notifications, and particularly refreshing a Realm which has change notifiers
+  registered.
```

Comment:
Same as above

Reply:
This actually shouldn't be in the changelog since it's fixing a regression we didn't ship.